hw3/React-App/src/Navbar.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./Navbar.css"; // import your custom CSS
+
+export default function Navbar() {
+  return (
+    <nav className="nav">

```

Comment:
Adding react-bootstrap would reduce the work with css file. Maybe you can consider adding it.

Reply:
yup bootstrap will be easier. I'll consider it.